custom_plugins/stream_overlays/static/css/main/trackdraw_map.css

```
@@ -0,0 +1,272 @@
+   ============================================================ */
+
+html {
+  background: transparent;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: var(--trackdraw-key-bg, grey);
+  font-family: "Barlow", sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Barlow" (font-family-name-quotes)

(font-family-name-quotes)